metabase/metabase
# Add time to event dates

Epic: https://github.com/metabase/metabase/issues/20289

Changes:
- Modifies `DateInput` to be only an Input
- Adds `NumericInput`
- Adds `TimeInput`
- Adds `DateSelector`
- Adds `DateWidget` = `DateInput` + `DateSelector`
- Adds time editing to events

How to test:
- Create or edit a timeline event
- Make sure it's possible to enter a time for a date
- Make sure it's saved properly and displayed in the list

Follow ups:
- Add am/pm
- Add timezone selection

<img width="667" alt="Screenshot 2022-02-21 at 20 03 53" src="https://user-images.githubusercontent.com/8542534/155010031-485639a4-5d8e-44df-a1e9-8eb8ebf18f67.png">
<img width="670" alt="Screenshot 2022-02-21 at 20 04 00" src="https://user-images.githubusercontent.com/8542534/155010038-da2aec3e-6ff8-43bc-aee8-e0dbfc5d4700.png">
<img width="662" alt="Screenshot 2022-02-21 at 20 04 10" src="https://user-images.githubusercontent.com/8542534/155010046-8322d9b0-23ad-41dd-8a0a-e3c5d06bcd45.png">
